config/routes.rb

```diff
@@ -1,6 +1,26 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  mount Rswag::Ui::Engine => '/api-docs'
+  mount Rswag::Api::Engine => '/api-docs'
```

Comment:
- [ ] These routes regarding project documentation are not working, because swagger helper does not exist in the spec folder.  Kindly fix it so docs can be displayed. 👍